horace_core/symop/@SymopRotation/SymopRotation.m

```diff
@@ -70,7 +70,9 @@
         % belong to the irreducible set in the upper right quadrant
 
             if ~exist('proj', 'var')
-                proj = line_proj();
+                proj = line_proj([1, 0, 0], [0, 1, 0], ...
+                                 'angdeg', [90, 90, 90], ...
+                                 'alatt', [1, 1, 1]);
             end
```

Comment:
would not mind this here, but if you do that, this means you are doing your symmetries in image coordinate system (so need to invent image) All previous symmetry transformation were performed in Pixel coordinate system and the result was transformed into real image coordinate system.